### PR TITLE
Add new Button Group Component

### PR DESCRIPTION
**Pull request by kristelaK**

### CODE REVIEW

File: src/components/inputs/ButtonGroup/ButtonGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+export const Default = Template.bind({});
+Default.args = {
+  children: <Typography> Default Button </Typography>
+};
+
+
+export const PrimaryButtonGroup = Template.bind({});
+PrimaryButtonGroup.args = {
+  children: <>
+    <Button>One</Button>
```
Comment (rep0x) [inline src/components/inputs/ButtonGroup/ButtonGroup.stories.tsx:72]: You can extract the JSX for the children into a constant and then just apply it here everywhere 👍

File: src/components/inputs/ButtonGroup/ButtonGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {
+  ButtonGroup as MuiButtonGroup,
+  ButtonGroupProps
+} from '@material-ui/core'
+
+export function ButtonGroup(props: ButtonGroupProps) {
+  return (
+    <MuiButtonGroup  {...props} />
+
```
Comment (rep0x) [inline src/components/inputs/ButtonGroup/ButtonGroup.tsx:9]: Remove empty line

**Review by rep0x — CHANGES_REQUESTED** [review]
Looks great, a few tiny little improvements 👍

**Comment by kristelaK** [comment]
@r3p0x I made the changes

**Review by rep0x — APPROVED** [review]

**Comment by rep0x** [comment]
Great good job

**Pull request merged**